yspryn/hw9/fraction.py

```diff
@@ -22,6 +22,8 @@ def convert(cls, item):
             return cls(item, 1)
         elif isinstance(item, cls):
             return item
+        else:
+            raise NotImplemented
```

Comment:
convert should not raise NotImplemented

and NotImplemented is not Exception, it will fail to raise this object